tests/testthat/test-validate_ensemble_inputs.R

```diff
@@ -71,3 +71,15 @@ test_that("error if models provide different output_type_ids", {
       validate_output_type_ids(task_id_cols = c("location", "horizon", "target", "target_date"))
   )
 })
+
+test_that("weights column already in model_outputs generates error", {
```

Comment:
let's update the name of the test case to ensure that it describes the contents of the test
```suggestion
test_that("weights depend on output_type_id for pmf output_type generates error", {
```

Reply:
Oops, thanks for catching that!